core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -18,8 +18,15 @@
 syntax = "proto3";
 package org.apache.spark.status.protobuf;
 
+/**
+ * Developer guides:
+ *   - Coding style: https://developers.google.com/protocol-buffers/docs/style
+ *   - Use int64 for job/stage IDs, in case of future extension in Spark core.
+ *   - Use `weakIntern` on string values in create new objects during deserialization.
+ */
+
 enum JobExecutionStatus {
-  UNSPECIFIED = 0;
+  JOB_EXECUTION_STATUS_UNSPECIFIED = 0;
```

Comment:
Is it more reasonable?

message Job {
   enum JobExecutionStatus {
      ...
   }
}

Reply:
OK, Let us follow code style(https://developers.google.com/protocol-buffers/docs/style#enums):
<img width="899" alt="image" src="https://user-images.githubusercontent.com/15246973/209942760-874bf1ab-9a00-421c-bcf9-6d8277c8b356.png">